src/app/Components/email-box/email-box.component.ts

```diff
@@ -1,15 +1,57 @@
 import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
 
 @Component({
   selector: 'app-email-box',
   templateUrl: './email-box.component.html',
-  styleUrls: ['./email-box.component.scss']
+  styleUrls: ['./email-box.component.scss'],
 })
 export class EmailBoxComponent implements OnInit {
-
-  constructor() { }
+  boxs = [
+    {
+      name: 'Inbox',
+      value: 'inbox',
+      open: false,
+    },
+    {
+      name: 'Finance',
+      value: 'finance',
+      open: false,
+    },
+    {
+      name: 'Travel',
+      value: 'travel',
+      open: false,
+    },
+    {
+      name: 'Personal',
+      value: 'personal',
+      open: false,
+    },
+    {
+      name: 'Drafts',
+      value: 'drafts',
+      open: false,
+    },
+    {
+      name: 'Sent',
+      value: 'sent',
+      open: false,
+    },
+  ];
+  active: string = 'inbox';
```

Comment:
for state with init value is a primitive value, just need to assign, no need to declare type.

Reply:
u haven't resolved this cmt.
`active: string = 'inbox';` >>> `active = 'inbox';`